Tests/Placement/readme.md

```diff
@@ -0,0 +1,32 @@
+These example applications interact with the placement system. Each example
```

Comment:
This Readme is missing details of how to use `run-all-tests.sh` (e.g. the required environment variable, where to execute, etc.) 🍠

Reply:
Added. It does say in the preamble of the script, but better to explain what the script is for in the readme.